### DISCUSSION TITLE
Badge on sidebar

### DISCUSSION THREAD

**Discussion by katanza12** [home-assistant/frontend#12014] [Dashboard]
I suggest to add the possiblity to manage badges on sidebar entities with a specific entry_number or counter, like actionable notifications does. 
For example, I have "HACS" on my sidebar menu and I would like to add a badge if there's (and how many) pending updates. Now I have a sensor for this, but a badge could be more immediate.
Thanks

**Comment by elchininet** [comment]
@katanza12,
If you refer to something like this:

![image](https://github.com/home-assistant/frontend/assets/3728220/d9d64fd1-7a5d-4fa6-a0d9-c15b200c5203)

Today has been released [a new feature](https://github.com/elchininet/custom-sidebar?tab=readme-ov-file#javascript-templates-example) in `custom-sidebar` that will allow you to achieve that.

Regards
